core/api/src/app/payments/send-lightning.ts

```diff
@@ -762,32 +765,53 @@ const executePaymentViaLn = async ({
       if (updateResult instanceof Error) {
         recordExceptionInCurrentSpan({ error: updateResult })
       }
+
+      await sendPaymentNotification({
```

Comment:
I added 2 new instances of sending notifications on pending (here) and failure lower down. Tbd if we want these (cc @dolcalmi)